roles/edpm_kernel/tasks/kernelargs.yml

```diff
@@ -15,8 +15,8 @@
 # under the License.
 
 - name: Get the command line args of the node
-  ansible.builtin.command: cat /proc/cmdline
-  register: cmdline
+  ansible.builtin.set_fact:
+    cmdline: "{{ lookup('ansible.builtin.file', '/proc/cmdline') }}"
 
 - name: TSX KernelArgs compute node reboot prevention
```

Comment:
side note this can all be removed.
TSX is no longer a thing and we should not need any of this logic anymore.
so that can be removed in a followup.

Reply:
:+1: